### ISSUE TITLE
Control buttons in the notification bar do not appear

### ISSUE THREAD

**Issue by Skayistheman** [TeamNewPipe/NewPipe#11051]
### Checklist

- [X] I am able to reproduce the bug with the latest version given here: [CLICK THIS LINK](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.27.0

### Steps to reproduce the bug

The problem appeared when I started using a Nubia Z50s pro with Version of android 13, MyOS 13.5.8 and kernel 5.15.104
![Screenshot_20240509_081642](https://github.com/TeamNewPipe/NewPipe/assets/139496558/87348c6d-3927-49a2-8186-053cac273677)

### Actual behavior

Controls do not appear in notification when screen is locked or unlocked

### Affected Android/Custom ROM version

Android 13/MyOS 13.5.8/kernel 5.15.104

### Affected device model

Nubia Z50s pro 

### Additional information

The problem appeared since I installed the app on the phone and I have not been able to correct it by changing the notification settings within the app or in the device settings

**Comment by VadosG** [comment]
Same problem on Nubia Neo 5G (Android 13/MyOS 13.0.3). But it worked fine on the previous version of NewPipe:

![Screenshot](https://github.com/TeamNewPipe/NewPipe/assets/132702182/4043e360-9fde-4cda-9faf-81dc77efb64e)

**Comment by Skayistheman** [comment]
It's true, In the previous version of the app it works correctly

**Comment by VadosG** [comment]
Yes

**Comment by keltar** [comment]
I have the same problem on Xiaomi 11 Lite 5G NE with android 13 - video thumbnail shows in notification, but no control buttons. 0.26.1 works fine.

**Comment by Feeeg** [comment]
Same issue，my device is iQOO 9 with Android 13， no control buttons above Newpipe version 0.26.1

**Comment by error-reporting** [comment]
seems to be android 13 issue?

**Comment by VadosG** [comment]
> seems to be android 13 issue?

No, because the developer of PipePipe has fixed this issue in their upcoming new version 5.0.0.
https://github.com/InfinityLoop1308/PipePipe
